integration

```diff
@@ -4,7 +4,6 @@
   "9a4gl/hass-peltec",
   "9rpp/securifi",
   "AaronDavidSchneider/SonosAlarm",
-  "abacao/hass_wibeee",
```

Comment:
This PR should not remove.
Removal needs to happen like this: <https://hacs.xyz/docs/publish/remove>

Reply:
Sure thing, I will kick off a separate PR for the removal.

Is there a recommended way to handle transitioning users from one integration to another? I was thinking both integrations could coexist in HACS default for a while and in the meantime I can work with @abacao to maybe do a new release just to be able to inform users of the upcoming changes in the release notes. Unless there's a better way?

Reply:
Let me know when there is a PR to remove my version

Reply:
Remove when you want (assuming old still works)

Reply:
Removal PR is https://github.com/hacs/default/pull/1242